plugins/rh-rbac-backend/src/permissions.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  createPermission,
+  ResourcePermission,
+} from '@backstage/plugin-permission-common';
+
+export const RESOURCE_TYPE_POLICY_ENTITY = 'policy-entity';
+
+/**
+ * Convenience type for permission entity
+ */
+export type PolicyEntityPermission = ResourcePermission<
+  typeof RESOURCE_TYPE_POLICY_ENTITY
+>;
+
+/**
+ * This permission is used to authorize actions that involve reading
+ * permission policies.
+ */
+export const policyEntityReadPermission = createPermission({
+  name: 'policy.entity.read',
+  attributes: {
+    action: 'read',
+  },
+  resourceType: RESOURCE_TYPE_POLICY_ENTITY,
+});
+
+/**
+ * This permission is used to authorize the creation of new permission policies.
+ */
+export const policyEntityCreatePermission = createPermission({
+  name: 'policy.entity.create',
+  attributes: {
+    action: 'create',
+  },
+  resourceType: RESOURCE_TYPE_POLICY_ENTITY,
+});
+
+/**
+ * This permission is used to authorize actions that involve removing permission
+ * policies.
+ */
+export const policyEntityDeletePermission = createPermission({
+  name: 'policy.entity.delete',
+  attributes: {
+    action: 'delete',
+  },
+  resourceType: RESOURCE_TYPE_POLICY_ENTITY,
+});
+
+/**
+ * This permission is used to authorize refreshing permission policies
+ */
+export const policyEntityRefreshPermission = createPermission({
```

Comment:
This is a confusing permission. If this is meant for re-reading the values from storage, I do not think we need an explicit permission for it (read should be enough). If this is meant for updating the values for existing keys on storage, in that case, the name refresh is not reflecting that purpose.

Reply:
I renamed it to policyEntityUpdatePermission to more accurately reflect that it is an update permission